### PR TITLE
[UwU] Update Post Cards to match Spec

### PR DESCRIPTION
**Pull request by crutchcorn**
This PR:

- Updates the card's styling to match the spec
- Updates all usages of the dense icon to include the intended padding
- Fixes the focus state of chips using `outline` rather than a border

image

### CODE REVIEW

File: src/components/post-card/post-card.module.scss [hunk 1]
```diff
@@ -1,112 +1,229 @@
...
-	&:hover {
-		background-color: var(--surface_primary_emphasis-low);
-		border-color: transparent;
-	}
+.authorList {
+	list-style: none;
+	margin: 0;
+	padding: 0;
+	display: flex;
+	flex-direction: row;
```
Comment (fennifith) [inline src/components/post-card/post-card.module.scss:110]:
```suggestion
	flex-direction: row;
	flex-wrap: wrap;
	gap: var(--card_detail-row_gap);
```

I believe this is needed to properly handle multiple authors in the list (currently they don't wrap, and have no spacing between authors).

File: src/components/post-card/post-card.module.scss [hunk 2]
```diff
@@ -1,112 +1,229 @@
...
-	margin-bottom: var(--spc-4x);
+.extendedPostContainer,
+.regularPostContainer {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+}
+
+.extendedPostImageContainer {
+	height: 1px;
```
Comment (fennifith) [inline src/components/post-card/post-card.module.scss:187]: This is still rendering the image on the mobile breakpoint with a height of `1px` (which is more visible in dark mode...) - I think this should have a `@include from($tabletLarge)` to make it `display: none;` on mobile.

File: src/components/post-card/post-card.module.scss [hunk 3]
```diff
@@ -1,112 +1,229 @@
...
+	height: 100%;
+	width: 100%;
+	position: absolute;
+	top: 0;
+	left: 0;
+	object-fit: cover;
+}
+
+.extendedPostImageBackground {
+	background-color: var(--primary_variant);
```
Comment (fennifith) [inline src/components/post-card/post-card.module.scss:226]: I think this background can be placed on `.extendedPostImageContainer` for the same result to save a div :)

(I did test this in-browser first)

**Review by PrattiDev — APPROVED** [review]
Solid work.

**Review by fennifith — APPROVED** [review]

**Pull request merged**